### PR TITLE
consul: minor fixes in docs

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY
Minor fixes in docs.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
plugins/modules/consul.py

### CODE REVIEW

**Comment by russoz** [comment]
Do not merge yet, may have found other documentation issues.

File: plugins/modules/consul.py [hunk 1]
```diff
@@ -78,76 +78,77 @@
...
         type: str
         description:
-          - the interval at which the service check will be run. This is a number
-            with a s or m suffix to signify the units of seconds or minutes e.g
-            C(15s) or C(1m). If no suffix is supplied, m will be used by default e.g.
-            C(1) will be C(1m). Required if the I(script) parameter is specified.
+          - The interval at which the service check will be run.
+            This is a number with a C(s) or C(m) suffix to signify the units of seconds or minutes e.g C(15s) or C(1m).
+            If no suffix is supplied C(s) will be used by default, e.g. C(10) will be C(10s).
+            Required if one of the parameters I(script), I(http), or I(tcp) is specified.
```
Comment (felixfontein) [inline plugins/modules/consul.py:106]:
```suggestion
          - Required if one of the parameters I(script), I(http), or I(tcp) is specified.
```
I think it's better to make this a new paragraph.

File: plugins/modules/consul.py [hunk 2]
```diff
@@ -78,76 +78,77 @@
...
         description:
-          - tags that will be attached to the service registration.
+          - Tags that will be attached to the service registration.
     script:
         type: str
         description:
-          - the script/command that will be run periodically to check the health
-            of the service. Scripts require I(interval) and vice versa.
+          - The script/command that will be run periodically to check the health of the service.
+            Requires I(interval) to be provided.
```
Comment (felixfontein) [inline plugins/modules/consul.py:99]:
```suggestion
          - Requires I(interval) to be provided.
```

File: plugins/modules/consul.py [hunk 3]
```diff
@@ -78,76 +78,77 @@
...
+            signify the units of seconds or minutes e.g C(15s) or C(1m).
+            If no suffix is supplied C(s) will be used by default, e.g. C(10) will be C(10s).
     tcp:
         type: str
         description:
           - Checks can be registered with a TCP port. This means that consul
             will check if the connection attempt to that port is successful (that is, the port is currently accepting connections).
             The format is C(host:port), for example C(localhost:80).
-            I(interval) must also be provided with this option.
+            Requires I(interval) to be provided.
```
Comment (felixfontein) [inline plugins/modules/consul.py:133]:
```suggestion
          - Requires I(interval) to be provided.
```

File: plugins/modules/consul.py [hunk 4]
```diff
@@ -78,76 +78,77 @@
...
         version_added: '1.3.0'
     http:
         type: str
         description:
-          - checks can be registered with an HTTP endpoint. This means that consul
+          - Checks can be registered with an HTTP endpoint. This means that consul
             will check that the http endpoint returns a successful HTTP status.
-            I(interval) must also be provided with this option.
+            Requires I(interval) to be provided.
```
Comment (felixfontein) [inline plugins/modules/consul.py:140]:
```suggestion
          - Requires I(interval) to be provided.
```

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks!

**Pull request merged**